src/main/java/org/folio/consortia/repository/entity/Tenant.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.consortia.repository.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import lombok.Data;
+
+@Entity
+@Data
+public class Tenant {
+  @Id
+  @GeneratedValue
+  @Column(name = "tenant_id", updatable = false, nullable = false, unique = true)
+  private String id;
```

Comment:
I think we can mark it with annotation @PrimaryKey instead of @Column and rename to tenantId

Reply:
Data JPA will look for ID field by default , i have tried customize it but did not work.